packages/components/src/List/Toolbar/ColumnChooserButton/hooks/columnChooserManager.hook.js

```diff
@@ -1,26 +1,7 @@
-import { useState } from 'react';
+import { useState, useMemo } from 'react';
 import flow from 'lodash/flow';
-import cloneDeep from 'lodash/cloneDeep';
 import { compareOrder } from '../service';
 
-const isItemVisible = item => item.visible;
-
-/**
- * Helps to change the select all visible status. If all columns are visible, select all is checked.
- * @param {array} columns
- */
-const isEveryItemVisible = columns => columns.every(isItemVisible);
-
-const findColumnLabel = label => column => column.label === label;
-
-export const getColumn = label => columns => columns.find(findColumnLabel(label));
-
-export const setColumn = (column, label) => columns => {
```

Comment:
it looks like a breaking change but they do not seems to be used outside. could you please confirm ?

Reply:
I couldn't find a place where it was used outside the components so I assume it's safe to remove it (don't know why it was exported prior this PR, tests ? 🤔)